data/signin.js

```diff
@@ -0,0 +1,265 @@
+class Signin{
+
+    constructor() {
+        this.selector_type = 'dusk';
+
+        this.params = {
+            group:{
+                count: null,
+                name: null,
+            }
+        }
+
+        this.selectors = {
+            email: 'signin-email_or_username',
+            password: 'signin-password',
+            submit: 'signin-signin',
+            page_heading: 'h3',
+            home_page_heading: 'h2',
+            page_subheading: 'p',
+            alert_box: 'alertdialog',
+            menu_item: 'menuitem',
+            logout_button: '=Logout',
+            eye_icon: 'eye',
```

Comment:
@satyajeet-s001 why we need  this selector?

Reply:
sir, for finding the alertbox which popups after entering the wrong credentials.

Reply:
sir if you are talking about attribute value "eye" , i have added one selector icon, as there is no unique attribute present to find the eye icon element.
refer: https://img-v4.getdemo.dev/screenshot/chrome_3cuPSs8ySd.mp4